articles/azure-vmware/manage-dhcp.md

```diff
@@ -130,6 +123,18 @@ If you want to send DHCP requests from your Azure VMware Solution VMs on the L2
 
    :::image type="content" source="media/manage-dhcp/edit-segment-security.png" alt-text="Screenshot of the Segment Security field" lightbox="media/manage-dhcp/edit-segment-security.png":::
 
+### To find your HCX L2 extended segment name
```

Comment:
Why is this section here?  You add a link under the note which implies these steps need to be the first set of steps before you continue.  If that's the case, I'd do this entire section like:

## Send DHCP requests to a non-NSX-T based DHCP server
If you want to send DHCP requests from your Azure VMware Solution VMs a non-NSX-T DHCP server, you'll create a new security segment profile.

>[!IMPORTANT]
>VMs on the same L2 segment that runs as DHCP servers are blocked from serving client requests.  Because of this, it's important to follow the steps in this section.

1. Locate the segment name of the L2 extension.

   1. Sign in to your on-premises vCenter, and under **Home**, select **HCX**.

   1. Select **Network Extension** under **Services**.

   1. Select the network extension you want to support DHCP requests from Azure VMware Solution to on-premises.

   1. Take note of the destination network name.

      :::image type="content" source="media/manage-dhcp/hcx-find-destination-network.png" alt-text="Screenshot of a network extension in VMware vSphere Client" lightbox="media/manage-dhcp/hcx-find-destination-network.png":::

1. In the Azure VMware Solution NSX-T Manager, select **Networking** > **Segments** > **Segment Profiles**.

1. Select **Add Segment Profile** and then **Segment Security**.

   :::image type="content" source="media/manage-dhcp/add-segment-profile.png" alt-text="Screenshot of how to add a segment profile in NSX-T" lightbox="media/manage-dhcp/add-segment-profile.png":::
1. Provide a name and a tag, and then set the **BPDU Filter** toggle to ON and all the DHCP toggles to OFF.

   :::image type="content" source="media/manage-dhcp/add-segment-profile-bpdu-filter-dhcp-options.png" alt-text="Screenshot showing the BPDU Filter toggled on and the DHCP toggles off" lightbox="media/manage-dhcp/add-segment-profile-bpdu-filter-dhcp-options.png":::
	
   :::image type="content" source="media/manage-dhcp/edit-segment-security.png" alt-text="Screenshot of the Segment Security field" lightbox="media/manage-dhcp/edit-segment-security.png":::


## Next steps
Learn more about [Host maintenance and lifecycle management](concepts-private-clouds-clusters.md#host-maintenance-and-lifecycle-management).

Reply:
Then it should be step #1 indicating that "if you need to..., then "this is how."  If you force them to the bottom of the page and then back up, it's a bad user experience!